kubernetes/samples/scripts/create-weblogic-domain/model-in-image/util-patch-restart-version.sh

```diff
@@ -0,0 +1,83 @@
+# !/bin/sh
```

Comment:
This can be done with just a few steps:
```bash
#! /bin/bash
current=`kubectl get domain sample-domain1 -n sample-domain1-ns -o=jsonpath='{.spec.restartVersion}'`
next=$((current + 1))
echo Rolling domain to restartVersion=$next
kubectl patch domain sample-domain1 -n sample-domain1-ns --type='json' -p='[{"op": "replace", "path": "/spec/restartVersion", "value": "'${next}'" }]'
```